packages/hooks/use-theme/src/index.ts

```
@@ -0,0 +1,93 @@
+        : ThemeProps.LIGHT;
+    }
+
+    return defaultTheme;
+  });
+
+  const setTheme = useCallback(
+    (newTheme: Theme) => {
+      try {
+        const targetTheme =
+          newTheme === ThemeProps.SYSTEM
+            ? window.matchMedia && window.matchMedia(MEDIA).matches
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)